godotsteam/godotsteam.cpp

```diff
@@ -6996,6 +7018,7 @@ String Steam::getQueryUGCTagDisplayName(uint64_t query_handle, uint32 index, uin
 		UGCQueryHandle_t handle = (uint64_t)query_handle;
 		SteamUGC()->GetQueryUGCTagDisplayName(handle, index, tag_index, tag, 256);
 	}
+	tag[255] = '\0';
```

Comment:
I tested that and looks that steam don't save normal data to 256 bytes of data and forgets about ending `\0`
This cause overflow, because `String tagDisplay = tag;` don't stop reading bytes, until it found `\0`